src/main/java/coffee/synyx/frontpage/plugin/feed/HtmlConverter.java

```diff
@@ -9,17 +8,30 @@ private HtmlConverter() {
         // ok
     }
 
-    static String toHtml(List<BlogEntry> entries) {
+    static String toHtml(Feed feed) {
         String html = "";
-        for (BlogEntry entry : entries) {
+
+        final FeedImage image = feed.getImage();
+        if (!Objects.equals(image.getUrl(), "")) {
+            html += "<a  href=\"" + image.getLink() + "\" " +
+                "title=\"" + image.getTitle() + "\" " +
+                "target=\"_blank\" " +
```

Comment:
Links in einem neuen Fenster öffnen hat paar Nachteile:
* accessibility
  * Link kann nicht mehr im selben Fenster geöffnet werden falls gewünscht. Ohne `_blank` kann der Anwender entscheiden ob selbes oder neues Fenster
  * when to use `target="_blank" `https://css-tricks.com/use-target_blank/
* performance
  * das neue Fenster wird im selben Prozess geöffnet: https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/
  * --> `rel="noopener"` dem link hinzufügen
* security
  * das neue Fenster hat per JavaScript Zugriff auf unsers: https://mathiasbynens.github.io/rel-noopener/
  * --> `rel="noopener"` dem link hinzufügen